poetry/packages/project_package.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, name, version, pretty_version=None):
         self.packages = []
         self.include = []
         self.exclude = []
+        self.data_files = {}
```

Comment:
why a `{}` vs `[]` as compared to https://github.com/sdispater/poetry/pull/539/files#diff-79da47ec0c1b34186a766622ab6e78ffR17